api/src/main/java/org/eclipse/microprofile/openapi/annotations/callbacks/Callback.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2017 SmartBear Software
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.openapi.annotations.callbacks;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.eclipse.microprofile.openapi.annotations.Operation;
+
+/**
+ * This object represents a callback URL that will be invoked.
+ **/
+@Target({ ElementType.FIELD,
+          ElementType.METHOD,
+          ElementType.PARAMETER,
+          ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+public @interface Callback {
```

Comment:
Is this an interceptor binding? If so, you should add the annotation with @InterceptorBinding so that the users know they need to treat it as an interceptor binding

Reply:
no, this one is just modelling the callback object: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#callbackObject